workspaces/shortcuts/plugins/shortcuts/src/api/ShortcutApi.ts

```diff
@@ -15,11 +15,11 @@
  */
 
 import { Shortcut } from '../types';
-import { createApiRef } from '@backstage/core-plugin-api';
+import { createApiRef, ApiRef } from '@backstage/frontend-plugin-api';
```

Comment:
This would likely break the current frontend system support, I don't think this change is needed.